transform_dialect/python/test_batch_matmul.py

```diff
@@ -0,0 +1,46 @@
+# Instructions
+#   Build IREE in `IREE_BUILD_DIR` with python bindings
+#   source ${IREE_BUILD_DIR}/.env && export PYTHONPATH
+#   export PATH=${PATH}:${IREE_BUILD_DIR}/tools
+#
+# Debug TD stuff by appending: --td-repro=1
+#
+# python test_batch_matmul.py
+
+import matmul_config as config
+import matmul_runner as runner
+import td_argparse
+
+problem_sizes = [
+   [8, 128, 128, 128],
+   # Currently fails inside IREE
+   # iree-compile: iree/compiler/src/iree/compiler/Codegen/Common/GPU/GPUPipelining.cpp:86:
+   # mlir::Operation *mlir::iree_compiler::replaceOpWithPredicatedOp(mlir::RewriterBase &, mlir::Operation *, mlir::Value):
+   # Assertion `false && "Unpeeled epilogue not supported with a side-effect instruction " "with no predication."' failed.
```

Comment:
You could add `, "peel": "1"` to your config.

https://github.com/iree-org/iree-samples/blob/main/transform_dialect/python/matmul_config.py#L171